Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V3.yml

```diff
@@ -271,6 +285,9 @@ outputs:
 - contextPath: File.SSDeep
   description: File SSDeep.
   type: string
+- contextPath: AcquiredFile
+  description: The acquired file details.
+  type: string
```

Comment:
That is from the PS Remote subplaybook
![image](https://github.com/demisto/content/assets/43602124/3fbefc48-22cd-498f-b9bb-2669d993cc74)

it seems like it selected String by default, but it's actually not a string. I'll fix.

Reply:
So maybe we should make it unknown then? cause string can be problematic when you use transformers - it will filter the options (unless you click on expand at the top).
TL;DR -  I think unknown in this case can be much better.

Reply:
Yeah fixed it